### PR TITLE
fix(actor-filter): add details for dynamic task assignment and actor …

### PR DESCRIPTION
**Pull request by redboul**
…filters

Closes [BS-15537](https://bonitasoft.atlassian.net/browse/BS-15537)

### CODE REVIEW

File: md/manage-users.md [hunk 1]
```diff
@@ -171,8 +171,11 @@ Next, it calls `getPossibleUsersOfPendingHumanTask` to get the list of users who
 The actor filter is applied when the task instance is created. This is known as **dynamic task assignment**. 
 The actor filter is applied and a shorter list of users is returned (in this case, just superExpert).
 
-Dynamic task assignment using `getPossibleUsersOfPendingHumanTask` re-evaluates the actor mapping including any actor filters. 
-It is useful if your organization changes after a process instance is started and you need to modify the list of users who can perform a task.
+Dynamic task assignment using `getPossibleUsersOfPendingHumanTask` re-evaluates the actor mapping.
+It is useful if your organization changes after a process instance is started and you need to modify the list of users who can perform a task
+
+If there is any Actor Filter specified for the given task, dynamic task assignment does not have any effect.  
+In such case, there is another [method](http://documentation.bonitasoft.com/javadoc/api/${varVersion}/org/bonitasoft/engine/api/ProcessRuntimeAPI.html#updateActorsOfUserTask(long)) that can be used as a separate call.
```
Comment (educhastenier) [inline md/manage-users.md]: link with ${varVersion} does not seem to work (on pr-doc at least)

Comment (educhastenier) [inline md/manage-users.md]: I would be even more explicit :
" ... as a separate call to force the re-execution of the actor filter and thus update the list of possible users"

Comment (redboul) [reply md/manage-users.md]: it works in http://doc.rd.lan:50122/?page=manage-users

Comment (educhastenier) [reply md/manage-users.md]: Yes, it works now. Dunno why it didn't the other day... :open_mouth:

**Comment by educhastenier** [comment]
Good to go to prod

**Pull request merged**